datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/transformers/count.py

```diff
@@ -0,0 +1,14 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def get_count(check, metric_name, modifiers):
+    count_method = check.count
+
+    def count(value, *, tags=None):
+        count_method(metric_name, value, tags=tags)
+
+    del check
+    del modifiers
```

Comment:
Just curious, why do these get deleted?

Reply:
to reduce memory usage of the cached closures